template.html

```diff
@@ -114,5 +114,17 @@
         </tr>
       </tbody>
     </table>
+    <script>
```

Comment:
Please Ignore this... Didn't have php to test so I used the html template and forgot to remove my changes